.github/workflows/test.yml

```diff
@@ -6,6 +6,9 @@ on:
       - main
       - 'release/**'
   pull_request:
+    paths-ignore:
+      - '**/*.md'
```

Comment:
I think you can just do '**.md' to account for all markdown files in the project

Reply:
Always confused about globs - where they work which way.
Pushing now.
Thanks!